pegasus/sites.v3/code.org/views/educate_box.haml

```diff
@@ -1,4 +1,5 @@
 -color ||= "#59b9dc"
+-item[:button_s] ||= 'Learn more'
```

Comment:
Roger, I just didn't (still don't) see where the `Learn more` button was previously.